src/applications/personalization/dashboard-2/components/IconCTALink.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+const IconCTALink = ({ CTA }) => {
+  const { ariaLabel, href, text, icon, onClick, boldText } = CTA;
+
+  const linkClass = `vads-u-text-decoration--none vads-u-padding-y--2p5 vads-u-padding-x--1 cta-link vads-u-font-weight--${
+    boldText ? 'bold' : 'normal'
+  }`;
+
+  return (
+    <a
+      aria-label={ariaLabel ? `${ariaLabel} (opens in new tab)` : ''}
+      href={href}
+      rel="noreferrer noopener"
+      target="_blank"
+      onClick={onClick || undefined}
+      className={linkClass}
+    >
+      <span className="fa-stack fa-sm vads-u-margin-right--1">
+        <i
```

Comment:
Stacking `icons` per [font awesome's documentation](https://fontawesome.com/how-to-use/on-the-web/styling/stacking-icons)